servers/visual/rasterizer_rd/shaders/scene_forward.glsl

```diff
@@ -757,45 +737,44 @@ void light_process_omni(uint idx, vec3 vertex, vec3 eye_vec, vec3 normal, vec3 a
 	}
 #endif //USE_NO_SHADOWS
 
-	light_compute(normal, normalize(light_rel_vec), eye_vec, color_specular.rgb, light_attenuation, albedo, roughness, metallic, specular,color_specular.a * p_blob_intensity,
+	light_compute(normal, normalize(light_rel_vec), eye_vec, color_specular.rgb, light_attenuation, albedo, roughness, metallic, specular, color_specular.a * p_blob_intensity,
 #ifdef LIGHT_TRANSMISSION_USED
-		transmission,
+			transmission,
 #endif
 #ifdef LIGHT_RIM_USED
-		rim * omni_attenuation, rim_tint,
+			rim * omni_attenuation, rim_tint,
 #endif
 #ifdef LIGHT_CLEARCOAT_USED
-		clearcoat, clearcoat_gloss,
+			clearcoat, clearcoat_gloss,
 #endif
 #ifdef LIGHT_ANISOTROPY_USED
-		binormal, tangent, anisotropy,
+			binormal, tangent, anisotropy,
 #endif
 #ifdef USE_SHADOW_TO_OPACITY
-		alpha
+			alpha,
```

Comment:
The missing comma was a bug here, toggling `USE_SHADOW_TO_OPACITY` would likely cause shader compilation failure.